portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -19,6 +20,7 @@ public Comment(long id, String title, String content, long timestamp) {
   private String convertMillisToTimestamp(long timestamp) {
     Date date = new Date(timestamp);
     SimpleDateFormat dateFormat = new SimpleDateFormat("MMM dd yyyy hh:mm a"); 
+    dateFormat.setTimeZone(TimeZone.getTimeZone("PST"));
```

Comment:
Any way to get local timezone based on IP?

Reply:
Resolved by moving computation from backend to frontend.